src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/TestLogAppender.kt

```diff
@@ -0,0 +1,44 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi
+
+import ch.qos.logback.classic.Level
+import ch.qos.logback.classic.spi.ILoggingEvent
+import ch.qos.logback.core.AppenderBase
+import java.util.concurrent.CopyOnWriteArrayList
+
+class TestLogAppender : AppenderBase<ILoggingEvent>() {
```

Comment:
This might look familiar 🤔 😉

Reply:
Nuke it! And the logback-test.xml file as well! 😁 
(see my comment above on the service class unit test)